### PR TITLE
feature/place-item_logic: implement some logic

### PR DESCRIPTION
**Pull request by K0sen**

### CODE REVIEW

File: frontend/src/pages/PlacePage.vue [hunk 1]
```diff
@@ -35,18 +43,35 @@ export default {
...
+
+    methods: {
+        tabChanged(activeTab) {
+            this.activeTab = activeTab;
+        }
+    },
+
     computed: {
-        ...mapState('place', ['currentPlace'])
+        ...mapState('place', ['place']),
```
Comment (nemoipaha) [inline frontend/src/pages/PlacePage.vue]: What the difference between 1-2  ...mapState('place', ['place/places']),

File: frontend/src/store/modules/place/actions.js [hunk 2]
```diff
@@ -2,22 +2,28 @@ import httpService from '@/services/common/httpService';
...
-            .then(function (response) {
-                commit('SET_CURRENT_PLACE', response.data.data);
-            })
-            .catch(function (err) {
-                console.log(err);
-            });
+        return new Promise((resolve, reject) => {
+            httpService.get('/places/' + id)
+                .then(function (response) {
+                    commit('SET_PLACE', response.data.data);
```
Comment (nemoipaha) [inline frontend/src/store/modules/place/actions.js]: No need for `commit('SET_PLACE', response.data.data);` here.  this state item unlikely will be reused in other components. You can do `this.place = response;` in component

File: frontend/src/store/modules/place/mutations.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export default {
-    SET_CURRENT_PLACE: (state, place) => {
-        state.currentPlace = place;
+    SET_PLACE: (state, place) => {
```
Comment (nemoipaha) [inline frontend/src/store/modules/place/mutations.js]: No need to share this state for now.

File: frontend/src/store/modules/place/state.js [hunk 4]
```diff
@@ -109,6 +109,6 @@ export default {
             ]
         }
     ],
-    truePlaces: null,
-    currentPlace: null
+    backendPlaces: null,
```
Comment (nemoipaha) [inline frontend/src/store/modules/place/state.js]: places / placeList is enough here

**Comment by K0sen** [comment]
responded to all comments

**Pull request merged**